crankshaft-engine/src/service/runner.rs

```diff
@@ -103,18 +115,14 @@ impl Runner {
 
         if backend.default_name() == "docker" && task.name.is_none() {
             let mut generator = self.name_generator.lock().unwrap();
-            // SAFETY: this generator should _never_ run out of entries.
             task.name = Some(generator.next().unwrap());
         }
 
         tokio::spawn(async move {
             let _permit = lock.acquire().await?;
-            let result = backend.clone().run(task, None, token)?.await;
 
-            // NOTE: if the send does not succeed, that is almost certainly
-            // because the receiver was dropped. That is a relatively standard
-            // practice if you don't specifically _want_ to keep a handle to the
-            // returned result, so we ignore any errors related to that.
```

Comment:
Likewise we can keep this comment and move the call to `run` above where it was before.